api/handler/object_list.go

```diff
@@ -121,32 +121,39 @@ func parseListObjectsArgsV2(r *http.Request) (*layer.ListObjectsParamsV2, error)
 	}
 	res.ListObjectsParamsCommon = *common
 
-	res.ContinuationToken = r.URL.Query().Get("continuation-token")
-	res.StartAfter = r.URL.Query().Get("start-after")
-	res.FetchOwner, _ = strconv.ParseBool(r.URL.Query().Get("fetch-owner"))
+	if val, ok := queryValues["continuation-token"]; ok {
```

Comment:
Maybe we should use `queryValues[""]` everywhere in this function?

Reply:
I'm not sure I understand you correctly, but where should I use queryValues[""] in the function. I replaced parsing everywhere.

Reply:
I mean we use `queryValues` differently: `queryValues.Get("start-after")` or `queryValues["continuation-token"]`. Question was, should we use the same style?

Reply:
No, we shouldn't. 
At least because `r.Url.Query` returns `Values` type which is an alias of  `map[string][]string`. If we want to use queryValues as a map, we will have to append `[0]` everywhere, I mean `queryValues[""][0]`. I'm not sure it looks good.